ext/EnzymeExt.jl

```diff
@@ -150,7 +126,7 @@ module EnzymeExt
         args2 = ntuple(Val(N)) do i
             Base.@_inline_meta
             if args[i] isa Active
-                Duplicated(Ref(args[i].val), arg_refs[i])
+                EnzymeCore.MixedDuplicated(args[i].val, arg_refs[i])
```

Comment:
This is correct for the CPU, but I think we won't be able to pass `MixedDuplicated` to the GPU. 
Or at least we need an Adapt rule.

Reply:
Do we not have those already like we do for the other annotations?

Reply:
Apparently not, now we do here: https://github.com/EnzymeAD/Enzyme.jl/pull/1551

Reply:
We should double check that this adapts to what we want on CUDA. I don't know if CuRef is right here...

Reply:
Sure but this is code wouldn’t affect any currently landed cuda support (this is only reverse mode).

So we should likely explore that in michel’s PR?

Reply:
And what about non CUDA GPUs?